constrained-voucher.mkd

```diff
@@ -362,13 +360,41 @@ it provisionally accepts this CA certificate as if it were the result of "CA cer
 
 4. If the trust anchor CA was NOT used to sign its LDevID, the Pledge MUST perform an actual "CA certificates request" (/crts) to the EST server to obtain the EST CA trust anchor(s) since these differ from the (temporary) pinned domain CA.
 
-5. When doing this /crts request, the Pledge MAY use a CoAP Accept Option with value TBD287 ("application/pkix-cert") to limit the number of returned EST CA trust anchors to only one.
+5. When doing this /crts request, the Pledge MAY use a CoAP Accept Option with value TBD287 ("application/pkix-cert") to limit the number of returned EST CA trust anchors to only one. 
+A constrained Pledge MAY support only this format in a /crts response, per {{Section 5.3 of I-D.ietf-ace-coap-est}}. 
 
 7. If the Pledge cannot obtain the single CA certificate or the finally validated CA certificate cannot be chained to the LDevID, then the Pledge MUST abort the enrollment process and report the error using the enrollment status telemetry (/es).
 
-<!-- ******************************************************************** -->
 
-### Registrar Extensions
+### EST-client Extensions {#brski-est-extensions-estclient}
+
+This section defines extensions to EST-coaps clients, used after the BRSKI bootstrap procedure is completed.
+(Note that such client is not called "Pledge" in this section, since it is already enrolled into the domain.)
+A constrained EST-coaps client MAY support only the Content-Format TBD287 ("application/pkix-cert") in a /crts response, per {{Section 5.3 of I-D.ietf-ace-coap-est}}. 
+
+In this case, it can only store one trust anchor of the domain. Although this is not an issue in case the domain trust anchor remains stable, special consideration is 
+needed for cases where the domain trust anchor can change over time. Such a change may happen during EST re-enrollment: typically, a change of domain CA requires all devices 
+operating under the old domain CA to acquire a new LDevID issued by the new domain CA. A client's re-enrollment may be triggered by various events, such as imminent expiry of its LDevID. 
```

Comment:
```suggestion
needed for cases where the domain trust anchor can change over time. 
{{RFC
Such a change may happen during EST re-enrollment: typically, a change of domain CA requires all devices 
operating under the old domain CA to acquire a new LDevID issued by the new domain CA. A client's re-enrollment may be triggered by various events, such as imminent expiry of its LDevID. 
```

Reply:
I spent awhile looking for the RFC on trust anchor replacement.  I know that I wrote an email about that in the past year to lamps, but I can't find it.